tests/cmake_integration/Nested_IDL/CMakeLists_duplicate_filename.txt

```diff
@@ -0,0 +1,50 @@
+# Distributed under the OpenDDS License. See accompanying LICENSE
```

Comment:
I don't think this file is going to be picked up by CMake like this is set up now.

Reply:
I'll remove this file.